app/controllers/buyers/applications_controller.rb

```diff
@@ -24,19 +22,17 @@ def index
     activate_menu :audience, :applications, :listing
 
     @states = Cinstance.allowed_states.collect(&:to_s).sort
-    accessible_services = current_account.accessible_services
-    @services = accessible_services.includes(:application_plans)
+    @services = accessible_services
```

Comment:
Is `@services` used in the view? In this case why not making an helper in the view? `method_helper :accessible_services`
That way we explicitly tell in the view that it is about accessible_services and not any services.
So the view cannot be used by anything else though it will break SRP ...
Dunno as you wish